src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs

```diff
@@ -1,84 +1,84 @@
-﻿using Microsoft.AspNetCore.Components;
-using Microsoft.AspNetCore.Components.Web;
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
 
 namespace Bit.Client.Web.BlazorUI
 {
-    public partial class BitMessageBar
+    public partial class BitMessageBar : BitComponentBase
```

Comment:
this is a partial class for BitMessageBar.razor and we have already inherited from BitComponentBase in there so there is no need to inherit again